cmake/mcuboot.cmake

```diff
@@ -103,6 +103,18 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   set(app_sign_depends
     $<IF:${sign_merged},${merged_hex_file_depends},zephyr_final>)
 
+  if (NOT DEFINED CONFIG_BOOT_SIGNATURE_KEY_FILE)
+    include(${CMAKE_BINARY_DIR}/mcuboot/shared_vars.cmake)
+    set(CONFIG_BOOT_SIGNATURE_KEY_FILE ${mcuboot_SIGNATURE_KEY_FILE})
+    message("
+    ---------------------------------------------------------
+    INFO: CONFIG_BOOT_SIGNATURE_KEY_FILE set from child iamge to:
```

Comment:
typo: `iamge` -> `image`

Reply:
I had corrected this typo locally, but forgotten to save before push.